bank.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'account'
+# SauceBank
+class Bank
+  attr_accessor :accounts
+  def initialize
+    @accounts ||= load_accounts
+  end
+
+  def load_accounts
+    []
+  end
+
+  def find_account(name)
+    @accounts.select do |acct|
+      acct.owner.name.downcase == name.downcase
```

Comment:
Having to go through the account, and owner to get the name feels like a small.  Typically if I have to call more that one "." I check to see if that level of abstraction is necessary.  

I think `account.name` would be better.